ironfish/src/network/peers/peerConnectionManager.ts

```diff
@@ -104,15 +104,6 @@ export class PeerConnectionManager {
       }
     }
 
-    if (connectAttempts < CONNECT_ATTEMPTS_MAX && this.peerManager.canCreateNewConnections()) {
-      const peer = this.peerManager.createRandomDisconnectedPeer()
-      if (peer && this.connectToEligiblePeers(peer)) {
-        connectAttempts++
-      } else if (peer) {
-        this.peerManager.tryDisposePeer(peer)
-      }
-    }
-
```

Comment:
this is replaced with the logic in the peerNetworks. Address manager is now a store of quality candidates rather than any candidate. These candidates are connected to at node startup